### PR TITLE
Create LICENSE.txt

### PR DESCRIPTION
**Pull request by FaustXVI**

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (vdemeester) [inline LICENSE.txt]: You should change this line though 👼 😛

**Review by vdemeester — CHANGES_REQUESTED** [review]
A nit 😄

File: LICENSE.txt (hunk 1 above)
Comment (FaustXVI) [reply LICENSE.txt]: Woops :p

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**